user-interface/frontend/themes/datamanager/components/all.css

```diff
@@ -363,11 +367,11 @@ Buttons
   background-color: var(--lumo-primary-color);
 }
 
-.button-size-medium-dialog {
+.button-size-dialog-medium {
```

Comment:
Why are those classes required? Wouldn't a selector checking for the `.button` class in a dialog be more streight forward (e.g. `.dialog-medium .button`, `.dialog-large .button` (https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) ? Where is this selector used? I cannot find any occurrence.

Reply:
I adjusted the css class to the already existing naming schema but forgot to update the string within the java class using the css class. Good point i adjusted it.